tutorials/detachable_joints.md

```diff
@@ -7,6 +7,7 @@ models. Because the system uses joints to connect models, the resulting
 kinematic topology has to be a tree, i.e., kinematic loops are not currently
 supported. This affects the choice of the parent link, and therefore, the
 parent model, which is the model that contains the `DetachableJoint` system.
+Once detached, the joint can be re-attached by publishing to a topic.
```

Comment:
Can you add a sentence here that when attaching a back a detachable joint, the object will be attached to the parent at it's current pose i.e, it may not go back to the original pose/configuration it had when it was originally attached.

Reply:
I wasn't sure what you were referring to `it` in the `it may not go back to the original pose/configuration it had when it was originally attached`. So let me know if you'd like me to make any modifications to the recent change shown here

Reply:
It's perfect! Thanks!